static/templates/stream_settings/stream_types.hbs

```diff
@@ -80,3 +80,16 @@
     </div>
 </div>
 {{/if}}
+
+{{#if (or is_admin is_stream_edit)}}
+    <div id="push-notifications-enabled">
+        <label class="checkbox">
+            <input type="checkbox" id="id_push_notifications_enabled" data-setting-widget-type="push_notifications_enabled" class="prop-element" name="push_notifications_enabled" {{#if push_notifications_enabled}}checked="checked"{{/if}}/>
+            <span></span>
+            {{t "Mobile push notifications enabled by default" }}
+        </label>
+        <i class="fa fa-exclamation-circle settings-info-icon {{#if realm_push_notifications_enabled}}hide{{/if}} tippy-zulip-tooltip"
```

Comment:
We can instead show this tooltip while hovering on the input and label as we did for `Deactivate organization` button as we would probably disable this setting if server is not configured to send push notifications. Or we can atleast use `fa-info-circle` so that we can keep it consistent with other settings pages. Though some of the places still use `fa-question-circle`.

Reply:
I have done the changes. Now tooltip is shown on hovering the whole input and label.